Riot/Modules/Room/TimelineCells/BaseRoomCell/RoomCellContentView.swift

```diff
@@ -172,7 +172,7 @@ final class BubbleCellContentView: UIView, NibLoadable {
 }
 
 // MARK: - BubbleCellReadReceiptsDisplayable
```

Comment:
Do all of the `MARK:`s need updating too?

Reply:
Indeed, updated